koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -29,6 +31,13 @@
 
 LOG = logging.getLogger(__name__)
 
+GCP_PARTITION_MAP = {
+    "source": "varchar",
+    "year": "varchar",
+    "month": "varchar",
+    "day": "varchar",
+}
```

Comment:
Global static values like this are generally better as an Enum. The interface is a bit different than a mapping, but Enums are immutable, so you don't have to worry about something changing it.
```suggestion
class GCPPartions(Enum):
    source = "varchar"
    year = "varchar"
    month = "varchar"
    day = "varchar"
```

Reply:
I don't think an Enum would be appropriate here, an enum having the same value is treated as an alias to the original definition so the items here don't behave as they should. Example from shell:
```
>>> class GCPPartitions(Enum):
...     source = "varchar"
...     year = "varchar"
...     month = "varchar"
...     day = "varchar"
... 
>>> ",".join([f"{item.name} {item.value} " for item in list(GCPPartitions)])
'source varchar '
>>> GCPPartitions.month
<GCPPartitions.source: 'varchar'>
```

Reply:
Ah, that's true. I was thinking "varchar" was some sort of placeholder for the real value that would be changed in a later commit. I don't know what the actual data looks like.

Reply:
we are using this to build the SQL statement that is creating the table in case it doesn't exist for a new customer or we drop it and have to recreate it. The actual data processing and insertion is separate from what this is being used for.